app/scripts/modules/core/src/deploymentStrategy/strategies/rollingredblack/AdditionalFields.tsx

```diff
@@ -103,7 +103,7 @@ export class AdditionalFields extends React.Component<IRollingRedBlackStrategyAd
         <div className="col-md-12 form-inline">
           <label>
             Wait
-            <HelpField content="Time to wait before disabling instances in old server group" />
+            <HelpField content="Time to wait before disabling instances in all old server groups" />
```

Comment:
is the behavior of these options meaningfully different enough that it's valuable to say 'instances in'? the wording is a little awkward and adding 'in this cluster' felt like it would make the sentence even more awkward, so i left that part out for RRB.

Reply:
Would it make sense to put these strings in `core/src/help/help.contents.ts` while this is being changed?